docs/teams.md

```diff
@@ -0,0 +1,30 @@
+# Teams with Gimbal
+
+A key feature of Gimbal is team management. The idea is enable teams to configure and define their own Ingress resources without requiring an administrator to assist. To enable this, users will be isolated to one or more namespaces and should have capabilities to create Ingress routes and also view Services and Endpoints within their respective namespace.
+
+## RBAC Rules 
+
+A key component of any secure Kubernetes cluster are permissions implemented via Role-Based Access Control (RBAC). Following is a sample RBAC `ClusterRole` which can be assigned to users within a team namespace:
```

Comment:
I think it'd be nice to call out that this is to be applied to the Gimbal cluster. At first it threw me off, and I started to think: "But how are they going to create services?" until I realized that these rules are for the Gimbal cluster.